deepchecks/checks/distribution/train_test_label_drift.py

```diff
@@ -0,0 +1,230 @@
+    For categorical columns, we use the Population Stability Index (PSI).
+    See https://en.wikipedia.org/wiki/Wasserstein_metric.
+
+
+    Args:
+        max_num_categories (int):
+            Only for categorical columns. Max number of allowed categories. If there are more,
+            they are binned into an "Other" category. If max_num_categories=None, there is no limit. This limit applies
+            for both drift calculation and for distribution plots.
+    """
+
+    def __init__(
+            self,
+            max_num_categories: int = 10
+    ):
+        super().__init__()
+        self.max_num_categories = max_num_categories
+
+    def run(self, train_dataset, test_dataset, model=None) -> CheckResult:
+        """Run check.
+
+        Args:
+            train_dataset (Dataset): The training dataset object.
+            test_dataset (Dataset): The test dataset object.
+            model: not used in this check.
+
+        Returns:
+            CheckResult:
+                value: dictionary of column name to drift score.
+                display: distribution graph for each column, comparing the train and test distributions.
+
+        Raises:
+            DeepchecksValueError: If the object is not a Dataset or DataFrame instance
+        """
+        return self._calc_drift(train_dataset, test_dataset)
+
+    def _calc_drift(
+            self,
+            train_dataset: Dataset,
+            test_dataset: Dataset,
+    ) -> CheckResult:
+        """
+        Calculate drift for all columns.
+
+        Args:
+            train_dataset (Dataset): The training dataset object. Must contain a label.
+            test_dataset (Dataset): The test dataset object. Must contain a label.
+
+        Returns:
+            CheckResult:
+                value: drift score.
+                display: label distribution graph, comparing the train and test distributions.
+        """
+        train_dataset = Dataset.validate_dataset(train_dataset)
+        test_dataset = Dataset.validate_dataset(test_dataset)
+
+        drift_score, method, display = self._calc_drift_per_column(
+            train_column=train_dataset.label_col,
+            test_column=test_dataset.label_col,
+            column_name=train_dataset.label_name,
+            column_type='categorical' if train_dataset.label_type == 'classification_label' else 'numerical',
+        )
+
+        headnote = """<span>
+            The Drift score is a measure for the difference between two distributions, in this check - the test
+            and train distributions.<br> The check shows the drift score and distributions for the label.
+        </span>"""
+
+        displays = [headnote, display]
+        values_dict = {'Drift score': drift_score, 'Method': method}
+
+        return CheckResult(value=values_dict, display=displays, header='Train Test Label Drift')
+
+    def _calc_drift_per_column(self, train_column: pd.Series, test_column: pd.Series, column_name: Hashable,
```

Comment:
Code duplication with regular drift check. Should be util or base class.

Reply:
Actually, not completely duplicated as the feature importance thingy is in there. 
Thought about consolidating and splitting to smaller functions that can be reused, but seems too complicated for now. I suggest we open a ticket for it for later.

Reply:
yeah you can open an issue for that